src/SWP/Bundle/ContentBundle/Doctrine/ODM/PHPCR/ArticleRepository.php

```diff
@@ -47,24 +47,26 @@ public function findAllArticles()
      *
      * @throws \Exception
      */
-    public function getQueryForRouteArticles(string $identifier, array $order) : SqlQuery
+    public function getQueryForRouteArticles(string $identifier, array $order = []) : SqlQuery
     {
-        $queryStr = sprintf("SELECT * FROM nt:unstructured as S WHERE S.phpcr:class='%s' AND S.route=%s AND S.status=published", Article::class, $identifier);
-        $allowedOrders = ['ASC', 'DESC'];
-        if (count($order) !== 2 || !in_array(strtoupper($order[1]), $allowedOrders)) {
-            throw new \Exception('Order filter must have two parameters with second one asc or desc, e.g. order(id, desc)');
-        }
+        $queryStr = sprintf("SELECT S.route FROM nt:unstructured as S WHERE S.route=%s AND S.status=published", $identifier);
```

Comment:
I know it was like that earlier but it's a good time to use `ArticleInterface::STATUS_PUBLISHED` instead of hardcoding this value directly here.